### PR TITLE
Adding Tabbed browsing interface

### PR DESCRIPTION
**Pull request by brvarner**
Created photo feeds, and added list group to access follower/leader pages.

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
             Resources
           </a>
           <div class="dropdown-menu">
-            <a class="dropdown-item" href="/photos">Photos</a>
+            <a class="dropdown-item" href="/#{current_user}/photos">Photos</a>
```
Comment (AdolfoNava) [inline app/views/shared/_navbar.html.erb]: you have to correct #{current_user} => <%=current_user.username%>

**Review by AdolfoNava — CHANGES_REQUESTED** [review]
Make that one change I commented and you should be good to go

**Review by AdolfoNava — APPROVED** [review]
Perfect

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job, Brandon!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files. 

You can now merge these changes back to main.

**Comment by CarlosCastanedaDev** [comment]
Please submit the PR link in Canvas so I can grade it and you can move forward.

**Pull request merged**